integration_tests/pages/prisonerList/PrisonerListPage.ts

```diff
@@ -76,6 +76,35 @@ export default class PrisonerListPage extends Page {
     return this
   }
 
+  hasPaginationControlsDisplayed(): PrisonerListPage {
+    this.paginationControls().should('exist')
+    return this
+  }
+
+  setPaginationNumber(value: number): PrisonerListPage {
+    this.paginationControls().find(`li:nth-of-type(${value}) a`).click()
+    return this
+  }
+
+  hasPreviousLinkDisplayed(): PrisonerListPage {
+    this.paginationControls().find('li:first-of-type a').should('contain', 'Previous')
```

Comment:
Could you use your `paginationFirstLink` method here?
```
this.paginationFirstLink().should('contain', 'Previous')
```

Would that work? And then do the same thing for the next link displayed?

Reply:
Good spot, just realised that `paginationFirstLink` isn't even used currently - maybe I started this trail of thought and got lost!